probe-rs/src/bin/probe-rs/cmd/run/mod.rs

```diff
@@ -25,40 +28,46 @@ const RTT_RETRIES: usize = 10;
 
 #[derive(clap::Parser)]
 pub struct Cmd {
+    /// The path to the ELF file to flash and run
+    pub(crate) path: String,
+
+    /// Options only used when in normal run mode
     #[clap(flatten)]
-    pub(crate) probe_options: ProbeOptions,
+    pub(crate) run_options: NormalRunOptions,
+
+    // ---- General Options ahead ----
```

Comment:
I'd prefer moving this into a different struct, and flattening the into this one. Where do these `General Options` end? What's special about this section?